Adaptors/Amqp/src/ConnectionAmqp.cs

```diff
@@ -40,25 +40,24 @@ public class ConnectionAmqp : IConnectionAmqp
   private readonly AsyncLazy               connectionTask_;
   private readonly ILogger<ConnectionAmqp> logger_;
   private readonly QueueCommon.Amqp        options_;
+  private          Connection?             connection_;
   private          bool                    isInitialized_;
 
   public ConnectionAmqp(QueueCommon.Amqp        options,
                         ILogger<ConnectionAmqp> logger)
   {
     options_        = options;
     logger_         = logger;
-    connectionTask_ = new AsyncLazy(() => InitTask(this));
+    connectionTask_ = new AsyncLazy(() => InitTask());
```

Comment:
I think this works:
```suggestion
    connectionTask_ = new AsyncLazy(InitTask);
```

Reply:
I'm questionning the use of an AsyncLazy here as we do not have any means to pass a cancellation token the way code is written.